File: app/src/main/java/com/mklc/endlessrecyclerapp/data/network/ApiService.java
package com.mklc.endlessrecyclerapp.data.network;

import com.mklc.endlessrecyclerapp.data.model.network.User;

import java.util.List;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;

public interface ApiService {

    @GET("users")
    Call<List<User>> getUsers(@Query("page") int page);
}

File: remotelib/src/main/java/com/mklc/remote/api/ApiService.kt
package com.mklc.remote.api

import com.mklc.remote.data.model.response.UserResponse
import io.reactivex.rxjava3.core.Single
import retrofit2.http.GET
import retrofit2.http.Query

interface ApiService {

    @GET("/public/v2/users")
    fun loadUsers(): Single<List<UserResponse>>

    @GET("/public/v2/users")
    fun loadUsers(@Query("page") page: Int, @Query("per_page") perPage: Int = 10): Single<List<UserResponse>>

}